buildah-imagestreams/role-image-registry-ca.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: openshift-image-registry-tekton
+  namespace: openshift-image-registry
```

Comment:
fyi @nikhil-thomas - @vdemeester suggested I point you to this piece and how https://github.com/gabemontero/obu uses it in this example.

Getter the cert from this configmap allows us to push to the OpenShift internal registry with TLS

In OpenShift today, when a v1 Build is started, we copy this config map into the local project.  That and the builder SA created on project creation, facilitates pushing to the internal image registry with TLS.

As I reference in the README, I've been told you did the changes to the pipelines operator so that it seeds th `pipeline` SA with the internal registry auth, much like is done for the `builder` SA.

Possibly though, the operator could copy the registry cert CM into a local CM copy in the namespace as well.

If that happened, then code like https://github.com/gabemontero/obu/blob/master/pkg/cmd/cli/cmd/internal_registry.go#L37-L47 would not need this cluster role to access the image registry namespace.

Reply:
It could just access this new CM that is in the local namespace.